docs/node-operators/migration-guides/upgrade-node.md

```diff
@@ -253,12 +255,12 @@ name = "genesis"
 
 [vega]
   [vega.binary]
-    path = "<VEGA-BIN>"
+    path = "vega"
```

Comment:
@fkondej - there's a variable at the top of the page for the vega binaries path. With that in mind, do you still think it should be changed?

> `<VEGA-BIN>`: the path to the Vega core binary, e.g., `/home/vega/bin/vega`

Reply:
It is up for discussion, but I don't think we should use `<VEGA-BIN>`.
We need to consider there will be more than one `vega` binary. E.g. for the `Protocol Upgrade`, there will be at least two: one before the upgrade and one after the upgrade.

Vegavisor proposes an approach that there should be a subdirectory for every vega version under `<VEGAVISOR-HOME>`. Each subdirectory is called the same as the version, where `genesis` is a special case. And each subdirectory contains the `vega` binary. And `run-config.toml`, i.e. the file for this example/snippet. When we put `vega` as value for the `path` parameter, we point to a binary called `vega` inside the same directory as `run-config.toml`.
On the other hand, `<VEGA-BIN>` starts with `/`, so it is an absolute path. Technically this can be correct, but does not prepare you there will be more than one `vega` binary.